site/src/examples/progress/HiddenLabel.tsx

```diff
@@ -0,0 +1,46 @@
+import { ReactElement, useState } from "react";
+import { CircularProgress, LinearProgress } from "@salt-ds/lab";
+import {
+  FlexItem,
+  FlexLayout,
+  FlowLayout,
+  RadioButton,
+  RadioButtonGroup,
+} from "@salt-ds/core";
+
+export const HiddenLabel = (): ReactElement => {
+  const [selectedType, setSelectedType] = useState("circular");
+
+  return (
+    <FlexLayout direction="column" style={{ height: "100%" }}>
+      <FlexItem>
+        <FlowLayout justify="center" className="controls" gap={1}>
+          <RadioButtonGroup direction="horizontal" defaultChecked>
```

Comment:
Is it worth doing this? Probably something to discuss with design.

This could easily be a Linear and Circular Progress side by side.

Reply:
I'll check it with Ben on Monday